From ae228b407c2a7cfa79ab8cce00c187b27dc029f0 Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@freescale.com>
Date: Thu, 16 Jun 2016 14:35:52 +0800
Subject: [PATCH 1098/5242] MLK-12907: ASoC: imx_mqs: mqs can't be recorder

commit  5c4611859d45c911b46ffed3dbac789ee50dcc7b from
https://source.codeaurora.org/external/imx/linux-imx.git

mqs can't be used as recorder, the capture property need to be zero.

Signed-off-by: Shengjiu Wang <shengjiu.wang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 sound/soc/fsl/imx-mqs.c |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/sound/soc/fsl/imx-mqs.c b/sound/soc/fsl/imx-mqs.c
index a3959d4..a09a52a 100644
--- a/sound/soc/fsl/imx-mqs.c
+++ b/sound/soc/fsl/imx-mqs.c
@@ -1,5 +1,5 @@
 /*
- * Copyright 2012, 2014-2015 Freescale Semiconductor, Inc.
+ * Copyright 2012, 2014-2016 Freescale Semiconductor, Inc.
  * Copyright 2012 Linaro Ltd.
  *
  * The code contained herein is licensed under the GNU General Public
@@ -119,7 +119,7 @@ static int be_hw_params_fixup(struct snd_soc_pcm_runtime *rtd,
 		.dynamic = 1,
 		.ignore_pmdown_time = 1,
 		.dpcm_playback = 1,
-		.dpcm_capture = 1,
+		.dpcm_capture = 0,
 	},
 	{
 		.name = "HiFi-ASRC-BE",
@@ -131,7 +131,7 @@ static int be_hw_params_fixup(struct snd_soc_pcm_runtime *rtd,
 		.no_pcm = 1,
 		.ignore_pmdown_time = 1,
 		.dpcm_playback = 1,
-		.dpcm_capture = 1,
+		.dpcm_capture = 0,
 		.ops = &imx_mqs_ops_be,
 		.be_hw_params_fixup = be_hw_params_fixup,
 	},
-- 
1.7.9.5

